src/Impostor.Api/Net/Messages/S2C/Message01JoinGameS2C.cs

```diff
@@ -18,7 +18,7 @@ public static void SerializeJoin(IMessageWriter writer, bool clear, int gameCode
             writer.Write(hostId);
             writer.Write(player.Client.Name);
             player.Client.PlatformSpecificData.Serialize(writer);
-            writer.WritePacked(player.Character?.PlayerInfo.PlayerLevel ?? 1);
+            writer.WritePacked(player.Character?.PlayerInfo?.PlayerLevel ?? 1);
```

Comment:
OUT OF SCOPE: above lines are duplicated in Msg07JoinedGameS2C. Refactor to move to a common location?